version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.1.2"
+version in ThisBuild := "1.1.3"
```

Comment:
Note to self: This should only be a version bump if the libthrift downgrade is needed.